build-container-image.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
Organizing the code into functions, would make it all more readable and one could reuse for example the check if a command exists.